pkg/deps/config/client.go

```diff
@@ -0,0 +1,81 @@
+package config
```

Comment:
What's the motivation between `client.go` as the name for this file?

Can you think of something better? E.g. even `tx_client.go` would be more detail.

Reply:
`tx_client_options.go`